macro_utils_h_generator/macro_utils.tt

```diff
@@ -26,290 +26,290 @@
 #include <string.h>
```

Comment:
>string [](start = 10, length = 6)

can you please add a #include <stddef.h> here too? As it is needed for the use of size_t

Reply:
Sure. Done.